### ISSUE TITLE
Bank sort does not work

### ISSUE THREAD

**Issue by Batchman85** [AdiAddons/AdiBags#414]
version 1.9.18 retail

when trying to sort bank, you get error item is locked.

**Comment by RainofTerra** [comment]
Same problem here.

**Comment by samlehman** [comment]
same

**Comment by axlmcc** [comment]
Confirming this same behavior. 

Initial inspection appears to indicate this bug is caused by the addition of the line `addon:CloseAllBags()` in widgets/ContainerFrame.lua from commit c7a4c11a which resolved issue #188 

```lua
function containerProto:CreateSortButton()
	self:CreateModuleButton(
		"S",
		10,
		function()
			-- addon:CloseAllBags()
			self.bagObject:Sort()
			self.forceLayout = true
		end,
		L["(Blizzard's) Sort items"]
	)
end
```

Removing that line fixes sorting while at the bank. However, I haven't confirmed whether or not it reintroduces the original error regarding the mailing of merged stacks.

**Comment by Talyrius** [comment]
> _**[SORTING BANK AND REAGENT BANK](https://us.forums.blizzard.com/en/wow/t/sorting-bank-and-reagent-bank/150660)**_
> _“Currently using GW2UI addon, which rolls the **`SortBankBags()`** and **`SortReagentBankBags()`** API calls into one button on the bank UI. When these two commands are called in succession, whether it be through the addon or even through a macro script, the second API call gets the ‘Item is Locked’ error message. The order of these two API calls does not matter, the second call always fails.”_
> _—**[Telanssra](https://worldofwarcraft.com/en-us/character/us/borean-tundra/telanssra)**_

Confirmed and fixed. This issue wasn't present at the time c7a4c11 was commited.
